system_tests/fast_confirm_test.go

```diff
@@ -40,9 +44,109 @@ import (
 	"github.com/offchainlabs/nitro/validator/valnode"
 )
 
+func TestFastConfirmationWithdrawal(t *testing.T) {
+	ctx, cancelCtx := context.WithCancel(context.Background())
+	defer cancelCtx()
+	builder, stakerA, cleanupBuilder, cleanupBackgroundTx := setupFastConfirmation(ctx, t)
+	defer cleanupBuilder()
+	defer cleanupBackgroundTx()
+
+	// Withdraw ETH from L2 to L1
+	gethhook.RequireHookedGeth()
```

Comment:
Why is this needed?
I tried running this test without this line and it worked fine 🤔

Reply:
yeah, it doesn't seem like its needed, I mainly copied it from some other test thinking it was needed.